TombEngine/Game/camera.cpp

```diff
@@ -60,6 +60,15 @@ Vector3i CamOldPos;
 CAMERA_INFO Camera;
 GameVector ForcedFixedCamera;
 int UseForcedFixedCamera;
+int LastAnglex;
```

Comment:
Keeping lots of globals is bad practice we are trying to avoid and eventually to get rid of it. I think you should try to wrap all fields for camera-mesh movement into new struct.

Additionally, keeping x/y/z values separately is also a bad practice, we have additional data type called `Vector3i` for storing integer vectors.

Reply:
I made a GameVector (x,y,z) LastAngle into a new OBJ_CAMERA_INFO struct